javascript/chronometer.js

```diff
@@ -1,34 +1,40 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+  this.currentTime = 0;
+  this.intervalId = null;
   }
 
   start(callback) {
-    // ... your code goes here
+  this.intervalId = setInterval(()=> {
+    if(callback){
+      callback();
+    }
+    this.currentTime++
+  } ,1000)
   }
 
   getMinutes() {
-    // ... your code goes here
+  return Math.floor(this.currentTime /60);
   }
-
+ 
   getSeconds() {
-    // ... your code goes here
+  return Math.floor(this.currentTime % 60)
   }
 
   computeTwoDigitNumber(value) {
     // ... your code goes here
   }
```

Comment:
Hi Jawid! One way to solve this one would be like this:
 
 computeTwoDigitNumber(value) {
    return ("0" + value).slice(-2);
  }
  
it will always add 0 and return only the last two values. so if you have 10 --> 010 --> slice(-2) --> 10, if that makes sense or 9 --> 09 --> slice(-2) --> 09.